client/me/payment-methods/add-credit-card/index.jsx

```diff
@@ -22,6 +24,11 @@ class AddCreditCard extends Component {
 		addStoredCard: PropTypes.func.isRequired
 	};
 
+	constructor( props ) {
+		super( props );
+		this.createPaygateToken = curry( createPaygateToken )( 'card_add' );
```

Comment:
Just curious, what is the benefit of curry over `createPaygateToken.bind( null, 'card_add' )` ?